Mathlib/ModelTheory/DirectLimit.lean

```diff
@@ -474,22 +474,22 @@ noncomputable def Equiv_iSup :
     simpa only [F, Embedding.codRestrict_apply', Subtype.mk.injEq]
   exact ⟨Equiv.ofBijective F ⟨F.injective, F_surj⟩, F.map_fun', F.map_rel'⟩
 
-theorem Equiv_isup_of_apply {i : ι} (x : S i) :
+theorem Equiv_iSup_of_apply {i : ι} (x : S i) :
     Equiv_iSup S (of L ι _ (fun _ _ h ↦ Substructure.inclusion (S.monotone h)) i x)
```

Comment:
Scope-creep, but `Equiv_iSup` is miscapitalised, right? Maybe not worth fixing the names in this PR if we'll have to fix them again very soon

Reply:
I though `iSup` was correct capitalization - is the `E` in `Equiv` supposed to be lowercase?

Reply:
I mean that `Equiv_iSup` is a `def` but is neither `UpperCamelCase` nor `lowerCamelCase`, even after disregarding the `iSup` exception

Reply:
I've tried to do appropriate camelCase, not sure what's best here.

Reply:
Mind making the rename a separate PR? Then we can easily get non-model theorists (not that I am one!) to review

Reply:
Fair enough - spelling reverted.